main.rb

```diff
@@ -0,0 +1,149 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+def create_student(age, name)
+  print('Has Parent Permission [y/n]:')
+  permission = gets.chomp
+  if permission == 'y'
+    Student.new(age, name)
+  else
+    Student.new(age, name, parent_permission: false)
+  end
+end
+
+def create_teacher(age, name)
+  print('Specialization:')
+  specialization = gets.chomp
+  Teacher.new(age, specialization, name)
```

Comment:
Kindly also improve upon these aspects of your UI, as can be seen in the original design:
- [ ] the text _age_ should begin with an **uppercase**.
- [ ] the text _title_ should begin with an **uppercase**.

**Typos:**
![image](https://user-images.githubusercontent.com/12745474/144579575-edab9591-e3e7-4ca0-ae82-e104cff1fff9.png)

Reply:
Fixed